topics/go/algorithms/data/bloom/doc.go

```diff
@@ -0,0 +1,23 @@
+/* Package bloom contains a simple implementation of a bloom filer.
+
+A Bloom filter is a space efficient probabilistic data structure.
+It supports two operations - adding a memeber and checking for membership.
+Asking for membership will return either "not in set" or "probably in set".
+
```

Comment:
Maybe a link to explain more. I've never used this and the idea of "probably in set" is confusing to me for practical purposes.